### ISSUE TITLE
Links in the Workflow Submissions widget should lead to the submission view, not the draft

### ISSUE THREAD

**Issue by MoritzLost** [engram-design/Workflow#169]
### What are you trying to do?

Not sure if this is a bug or feature request.

The plugin provides a dashboard widget `Workflow Submissions` that lists all submissions with a specific status. The most common use-case is probably to display a list of pending submissions on the dashboard. Currently, the links on the listed submissions lead directly to the submitted draft, not to the new submission details view. It would make more sense for this link to lead to the submission details view.

### What's your proposed solution?

Either have the link in the submissions widget go to the submission details view, or provide both links to both the details view and the draft. To keep it simple, I would prefer if only the submission details view was linked.

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/workflow` requirement in `composer.json` to:

```
"require": {
  "verbb/workflow": "dev-craft-4 as 2.0.2,
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [2.0.3](https://github.com/verbb/workflow/releases/tag/2.0.3)
